### ISSUE TITLE
Unable to unsend messages

### ISSUE THREAD

**Issue by Yodoma** [sindresorhus/caprine#2262]

**Comment by RaspberryKitty1** [comment]
you know providing no additional information is gonna help nobody. just saying.

**Comment by Yodoma** [comment]
I agree. For some reason, the text I wrote didn't go through. I said that whenever I unsend it doesn't work.

<img width="168" alt="Screenshot 2024-12-09 at 9 35 36 AM" src="https://github.com/user-attachments/assets/fa981753-2f93-4c51-a7bc-008527143075">
